sass/blocks/intro/_intro.scss

```diff
@@ -32,12 +43,31 @@
   &--smaller {
     height: 16rem;
     background: url("../img/text-sedona.svg"),
+      url("../img/bg-intro-triangle-mobile.svg"),
       url("../img/bg-back-mobile@1x.jpg");
     background-repeat: no-repeat,
+      no-repeat,
       no-repeat;
     background-size: 23.1rem,
+      100% 100%,
       auto;
     background-position: 49% 43%,
+      0 6.7rem,
       50% 100%;
+
+    @media (min-resolution: 144dpi), (min-resolution: 1.5dppx) {
+      background: url("../img/text-sedona.svg"),
+        url("../img/bg-intro-triangle-mobile.svg"),
+        url("../img/bg-back-mobile@2x.jpg");
+      background-repeat: no-repeat,
+        no-repeat,
+        no-repeat;
+      background-size: 23.1rem,
+        100% 100%,
+        auto;
+      background-position: 49% 43%,
```

Comment:
also, responsiveness doesn't look good on the triangle
![kapture 2018-02-09 at 19 16 54](https://user-images.githubusercontent.com/1570578/36045516-e4ed75e8-0dcd-11e8-8352-2c666123385c.gif)